vochain/keykeeper/keykeeper.go

```diff
@@ -210,7 +196,7 @@ func (k *KeyKeeper) OnProcess(pid, eid []byte, censusRoot, censusURI string, txi
 		return
 	}
 	// If keys already exist, do nothing (this happens on the start-up block replay)
-	if len(p.EncryptionPublicKeys[k.myIndex])+len(p.CommitmentKeys[k.myIndex]) > 0 {
+	if len(p.EncryptionPublicKeys[k.myIndex]) > 0 {
```

Comment:
```suggestion
	if len(p.EncryptionPublicKeys[k.myIndex]) > 0 { // Jordi, you might not know that you can directly suggest edits now :)
```

See the "insert suggestion" button, the left-most button on the top toolbar when you write a comment on a line.

Reply:
Meaning @jordipainan, of course

Reply:
@jordipainan so let's do it via edit suggestion :)

Reply:
hahah okii thanks!